openpype/tools/publisher/widgets/widgets.py

```diff
@@ -198,12 +205,24 @@ def __init__(self, parent=None):
         self.setLayoutDirection(QtCore.Qt.RightToLeft)
 
 
+class SaveBtn(PublishIconBtn):
+    """Save context and instances information."""
+    def __init__(self, parent=None):
+        icon_path = get_icon_path("save")
+        super(SaveBtn, self).__init__(icon_path, parent)
+        self.setToolTip(
+            "Save changes ({} + S)".format(get_control_str())
+        )
+
+
 class ResetBtn(PublishIconBtn):
     """Publish reset button."""
     def __init__(self, parent=None):
         icon_path = get_icon_path("refresh")
         super(ResetBtn, self).__init__(icon_path, parent)
-        self.setToolTip("Refresh publishing")
+        self.setToolTip(
+            "Refresh publishing ({} + S)".format(get_control_str())
+        )
```

Comment:
`+ S` is definitely a typo here. 


```suggestion
        self.setToolTip(
            "Reset & discard changes ({} + R)".format(get_control_str())
        )
```